about/contact.md

```diff
@@ -0,0 +1,11 @@
+# Contact Us
+
+Are you interested in learning more about OceanHackWeek, helping out with the next event, or learning about the next event and how to prepare a strong application? You can:
+
+- Look over this web site, both general pages and [past events](pasthackweeks)
+- Write to us at [oceanhkw@uw.edu](mailto:oceanhkw@uw.edu).
```

Comment:
```suggestion
- Write to us at [info@oceanhackweek.org](mailto:info@oceanhackweek.org).
```

While we're at it, how about we also change the email address associated with the top-right icons, as part of this PR?

Reply:
👍

Reply:
I've updated the email address both here and and in the top-right icon.